### PR TITLE
[WOR-562] Remove "Community Workbench" from Terra welcome message

### PR DESCRIPTION
**Pull request by cahrens**
This is the only place I found "Community Workbench" in the terra-ui codebase.

Before:

image

After:

image

### CODE REVIEW

**Comment by cahrens** [comment]
Posted in Doc Slack channel in case documentation or screen shots are impacted-- https://broadinstitute.slack.com/archives/CE99R8RD5/p1665775737341669

**Review by kyuksel — APPROVED** [review]

**Review by aherbst-broad — APPROVED** [review]

**Comment by cahrens** [comment]
Waiting on BVM sign-off.

**Comment by cahrens** [comment]
The powers that be have decided not to do this.

**Pull request closed without merging**
